sources/packages/web/src/services/http/InstitutionLocation.ts

```diff
@@ -24,19 +26,28 @@ export class InstitutionLocationApi extends HttpBaseClient {
 
   public async updateInstitutionLocation(
     locationId: number,
-    updateInstitutionLocationDto: InstitutionLocationFormAPIInDTO,
+    updateInstitutionLocationDto:
+      | InstitutionLocationPrimaryContactAPIInDTO
```

Comment:
is this one for the institution and one for aest. if so, I think this and the service method can  be 2 separate methods

Reply:
and I think this function is consumed separately too

Reply:
the calling URL is same with different payloads, so it it okay to have a single method to ignore code duplication